sdk/template/template/README-TEMPLATE.md

```diff
@@ -97,6 +97,24 @@ const { DefaultAzureCredential } = require("@azure/identity");
 const client = new TextAnalyticsClient("<endpoint>", new DefaultAzureCredential());
 ```
 
+### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+### CORS
+
+> note to package authors: If the service supports CORS please use this section to describe how to enable CORS, otherwise delete this section. See an example below from Azure Storage:
```

Comment:
If the service does not support CORS, shouldnt we call that out as well?

Reply:
Yeah, that would be good. I was hesitant because some of the discussion in the issue showed preference to "add a section on enabling cors if the service actually supports it". What do you think?

And, if we want to add it what do you think about:

> Note to package authors: If the service supports CORS please use this section to describe how to enable CORS, otherwise please replace this section with "Note: This package cannot be used in the browser due to {Service Name} service limitations, please refer to [this document](https://github.com/Azure/azure-sdk-for-js/blob/master/samples/cors/ts/README.md) for guidance"

Reply:
Can we make that document KV agnostic? Is the content applicable for other services so that we can add a comment saying "Though we use KV in this document, it should apply to any other service that does not support CORS"

Reply:
We can and we will! 😄